CHANGELOG.md

```diff
@@ -30,18 +24,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 	`oeapkman --optee exec gcc -c file.c` cross-compile `file.c` to target OP-TEE.
   - See [samples/apkman](samples/apkman) for a complete example demonstrating use of the `sqlite` database library within enclaves.
 - Support for `compiler-rt`. `oelibc` includes LLVM's `compiler-rt-10.0.1`.
-- Support FIPS-enabled OpenSSL based on [SymCrypt](https://github.com/Microsoft/SymCrypt)
+
+## Changed
+- Updated libcxx to version 10.0.1
+
+
+[v0.17.6][v0.17.6_log]
+--------------
+
+### Added
+- Added support FIPS-enabled OpenSSL based on [SymCrypt](https://github.com/Microsoft/SymCrypt)
```

Comment:
Nit: Added support `for` FIPS-enabled OpenSSL based on [SymCrypt](https://github.com/Microsoft/SymCrypt)`.`